internal/mockstatsd/clientinterface.go

```diff
@@ -0,0 +1,265 @@
+// Code generated by MockGen. DO NOT EDIT.
+// Source: vendor/github.com/DataDog/datadog-go/statsd/statsd.go
+//
+// Generated by this command:
+//
+//	mockgen -package mockstatsd -source vendor/github.com/DataDog/datadog-go/statsd/statsd.go -destination internal/mockstatsd/clientinterface.go -imports github.com/mozilla-services/autograph/vendor/github.com/DataDog/datadog-go/statsd=github.com/DataDog/datadog-go/statsd ClientInterface
```

Comment:
It looks like mockgen is long deprecated/archived - is there something maintained we ought to be using for this instead?

Reply:
Oh, you might be looking at the https://github.com/golang/mock one, but I've installed https://github.com/uber-go/mock here in the go.mod and as the mockgen binary locally. That one is more supported than the other.

Though, I do dream of just taking patreon money and just maintaining this thing better than both.

Reply:
Ah; my bad. Thanks for the pointer!